server/controllers/oneEvent.js

```diff
@@ -0,0 +1,33 @@
+const {
+  Event, Coffee, Lunch, Equipment,
+} = require('../database/models');
+
+exports.oneEvent = ((req, res, next) => {
+  const { id } = req.params;
+  Event.findOne(
+    { where: { id }, raw: true },
+  ).then((event) => {
+    Coffee.findOne(
+      { where: { eventId: id }, raw: true },
+    ).then((coffee) => {
+      Lunch.findOne(
+        { where: { eventId: id }, raw: true },
+      ).then((lunch) => {
+        Equipment.findOne(
+          { where: { eventId: id }, raw: true },
+        ).then((equipment) => {
+          if (!event || event.length === 0) {
```

Comment:
The result is an object and you used ```event.length``` but this thing uses when you have an array

Reply:
done removed :+1: